ocaml-ts-mode.el

```diff
@@ -295,7 +300,12 @@
      ;; signatures and misc
      (instance_variable_name) @font-lock-variable-name-face
      (value_specification (value_name) @font-lock-variable-name-face)
-     (external (value_name) @font-lock-variable-name-face))
+     (external (value_name) @font-lock-variable-name-face)
+     ;; assignment of bindings in various circumstances
+     (type_binding ["="] @font-lock-keyword-face)
+     (let_binding ["="] @font-lock-keyword-face)
+     (field_expression ["="] @font-lock-keyword-face)
+     (for_expression ["="] @font-lock-keyword-face))
```

Comment:
I'm not sure about this (or, actually, using the keyword face / feature for `->` like I did below). My thinking was: it's quite nice to visually separate patterns from function/match case bodies, and one could make the same argument for let-bindings. 

On the other hand, neither rust-ts-mode nor typescript-ts-mode highlight the punctuation in lambdas, let expressions, or match cases. Based on the emacs docs[0], it might make sense to add font-lock features for various kinds of punctuation at level 4 instead (using font-lock-punctuation-face), which users can easily toggle. How does that sound?

[0] https://www.gnu.org/software/emacs/manual/html_node/emacs/Parser_002dbased-Font-Lock.html

Reply:
My thinking here was that it might be helpful to highlight bindings to distinguish them from other operators, but I'd be happy to do without it. Bindings feel built-in, hence the keyword face. I'm a bit fuzzy on what "punctuation" is supposed to mean in general — I'm definitely in favour of keeping some of the more detailed font-locking for level 4.

Reply:
Sure, makes sense -- same reason I added highlighting for "->". I'm just pointing out that I intend to try make the font-lock rules more "standard" as far as I understand what the  treesit.el authors were going for, and that might mean changing these clauses.

Reply:
Yep, following any established conventions is a solid plan!